examples/gaussian_processes/gaussian_process.ipynb

```diff
@@ -4,18 +4,23 @@
    "cell_type": "markdown",
```

Comment:
<p>I have added it to [<a href="https://einstats.python.arviz.org/en/latest/" target="_blank">xarray-einstats</a> If you install de development version you'll be able to use it already. I would therefore use:</p><p> </p><pre class="ql-syntax" spellcheck="false">from xarray_einstats.stats import multivariate_normal

post = az.extract(gp_trace, num_samples=200)
y_pred = multivariate_normal.rvs(post["mu_post"], post["sigma_post"], dims=("pred_id", "pred_id2"))
</pre><p> </p><p>Both looping and np.vectorize are prefectly fine, but I find it very annoying to 1) need to do it every single time I want samples from a multivariate normal (doesn't really apply to this notebook, but will apply to reader's lives) and 2) loose all the labels and dimension information. Here I tried to see what the data represents in the original example and wasn't able to find it, hence the <code>pred_id</code> suggestions, but in general I do have information which makes it easier for me to work with named dimensions.</p><p> </p><p>Note: I think you will also need to install arviz development version to use <code>az.extract</code> in addition to einstats development version.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/404/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Modified! and thanks for fixing the issue @OriolAbril